tests/syft/lib/tenseal/tenseal_ckkstensor_test.py

```diff
@@ -0,0 +1,374 @@
+# third party
+import pytest
+
+# syft absolute
+import syft as sy
+
+from typing import Any
+
+
+def _almost_equal_number(v1: Any, v2: Any, m_pow_ten: int) -> bool:
```

Comment:
Q: Why don't use the ```math.isclose``` it has ```absolute``` and ```relative``` tolerance?

Reply:
Theres also approx from pytest.
`from pytest import approx`